### ISSUE TITLE
Write an intuitive manual for bubblesub

### ISSUE THREAD

**Issue by Luni-4** [bubblesub/bubblesub#61]
It would be useful to write an intuitive manual to explain how `bubblesub` should be used. For now there is only a markdown for shortcuts, but we could add other files that explain the various features and how to use the plugins.

**Comment by rr-** [comment]
That's a great idea and I approve it, but don't you think it's gonna be a double effort if there's a plan to change the views dramatically?

**Comment by Luni-4** [comment]
Yep, it's gonna be a double effort, so let's wait #18 to be closed then
